repo/packages/H/history/0/config.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+    "mesos/master/url": {
+      "description": "The URL of the Mesos master. The format is a comma-delimited list of URLs like http://host:port.",
+      "type": "string",
+      "default": "http://leader.mesos:5050"
+    },
+    "history/resource/cpus": {
+      "description": "CPU shares to allocate to the history service.",
+      "type": "number",
+      "minimum": 0.0,
+      "default": 0.1
+    },
+    "history/resource/mem": {
+      "description": "Memory (MB) to allocate to each history service instance.",
+      "type": "integer",
+      "minimum": 0,
+      "default": 256
+    },
+    "history/port": {
```

Comment:
also we have a standard of using property hierarchy and dot notation
 and we needed required to be populated.